airflow/utils/db.py

```diff
@@ -67,39 +67,45 @@
 
 log = logging.getLogger(__name__)
 
-REVISION_HEADS_MAP = {
+_REVISION_HEADS_MAP = {
+    "1.5.0": "338e90f54d61",
+    "1.5.2": "52d714495f0",
+    "1.6.0": "40e67319e3a9",
+    "1.6.2": "4446e08588",
+    "1.7.0": "1968acfc09e3",
+    "1.7.1": "2e82aab8ef20",
+    "1.8.2": "d2ae31099d61",
+    "1.10.0": "9635ae0956e7",
+    "1.10.2": "41f5f12752f8",
+    "1.10.3": "a56c9515abdc",
+    "1.10.4": "004c1210f153",
+    "1.10.7": "fe461863935f",
+    "1.10.5": "74effc47d867",
```

Comment:
```suggestion
    "1.10.5": "74effc47d867",
    "1.10.7": "fe461863935f",
```

Reply:
I think there's a bug. it's supposed to order this properly. checking...

Reply:
Found it, ordering by filenames will not work because of the way we added files in the migrations/revisions. File 0048_1_10_3 is v1.10.3 but file 0045_1_10_7 is v1.10.7. So I need to find another way...

Reply:
I have removed adding versions < 2.0.0, and it solves it instead of trying to resolve the naming of versions less than 2.0 which we don't support in the upgrades

Reply:
SMART! Remove the problem rather than try to solve it :)